Vizzuality/landgriffon
# Add role check to upload actual data

### General description

This PR adds a role check to enable the 'upload data' button  at  '/data' page.

### Testing instructions

1. Login with an admin account and go to '/data'. The 'Upload data source' button should be enabled.
<img width="1083" alt="Screenshot 2023-01-24 at 09 15 35" src="https://user-images.githubusercontent.com/48164343/214242368-601140e2-063a-4888-8fdb-192f9a0f3b82.png">

2. Logout and login with a user account. The 'Upload data source' button should be disabled.
<img width="1083" alt="Screenshot 2023-01-24 at 09 16 00" src="https://user-images.githubusercontent.com/48164343/214242395-bf910ad3-fc4a-470a-9778-f7d4f7fa91a3.png">


## Task 
[LANDGRIF-1124](https://vizzuality.atlassian.net/browse/LANDGRIF-1124)

## Checklist before merging



[LANDGRIF-1124]: https://vizzuality.atlassian.net/browse/LANDGRIF-1124?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
